### PR TITLE
Fix stack verflow

### PR DESCRIPTION
**Pull request by GuillaumeGomez**
cc @EPashkin

### CODE REVIEW

File: src/strip.rs [hunk 1]
```diff
@@ -273,6 +274,7 @@ fn build_event_inner(
                 event_list.push(
                     EventInfo::new(*line, EventType::Comment(b_content[*line].to_owned())));
                 move_to(&words, it, "\n", line);
+                println!("end of move: {}", words[*it]);
```
Comment (EPashkin) [inline src/strip.rs]: Debug print?

File: src/strip.rs [hunk 2]
```diff
@@ -368,15 +364,32 @@ fn build_event_inner(
                                                            line), " ")))));
             }
             "{" => {
-                par_count += 1;
+                println!("new one! {}", words[*it]);
```
Comment (EPashkin) [inline src/strip.rs]: Debug print?

**Comment by EPashkin** [comment]
@GuillaumeGomez Thanks, 👍

File: src/strip.rs (hunk 1 above)
Comment (GuillaumeGomez) [reply src/strip.rs]: Good catch!

File: src/strip.rs (hunk 2 above)
Comment (GuillaumeGomez) [reply src/strip.rs]: Good catch as well!

**Pull request merged**